packages/api-clients/open-api/m2mEventApi.yml

```
@@ -0,0 +1,881 @@
+openapi: 3.0.3
+info:
+  title: M2M Events Manager
+  description: HTTP service to manage M2M Events
+  version: "{{version}}"
+  contact:
+    name: API Support
+    url: http://www.example.com/support
+    email: support@example.com
+  termsOfService: "http://swagger.io/terms/"
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: "/"
+    description: HTTP service to manage M2M Events
+security:
+  - bearerAuth: []
+tags:
+  - name: m2mEvents
+    description: M2M Events routes
+  - name: health
+    description: Verify service status
+paths:
+  /eservices:
+    get:
+      tags:
+        - m2mEvents
+      summary: Get paginated E-Service M2M Events
+      description: Returns a list of E-Service M2M Events
+      operationId: getEServiceM2MEvents
+      parameters:
+        - $ref: '#/components/parameters/LastEventId'
+        - $ref: '#/components/parameters/Limit'
+      responses:
+        "200":
+          description: E-Service M2M events retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EServiceM2MEvents"
+        "400":
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "401":
+          description: Unauthorized
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /agreements:
+    get:
+      tags:
+        - m2mEvents
+      summary: Get paginated Agreement M2M Events
```

Sorry, one last thing 🙏  Let's remove the word "paginated" for what we discussed in previous comments

```suggestion
      summary: Get Agreement M2M Events
```